### ISSUE TITLE
txa ant addon failing

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#2816]
We have a user report that the txa ant addon is failing in Bforartists. It works fine in Blender 3.0 and 3.1. But in Bforartists it simply fails.

I had already a look at it twice. The errors leads to nirvana. It is at least nothing that i could identify. My first thought was that it is because we are already at 3.1 because we rely at the master. But as already stated, the addon also works fine with the newest blender build.

Ideas?

https://www.bforartists.de/forums/topic/txa-landscape-addon-throws-errors-in-bforartists-3-0/#post-24123

https://github.com/nerk987/txa_ant

![antaddon](https://user-images.githubusercontent.com/13919702/145252193-12a93813-f5a8-45e6-9e87-1d0de50b39a9.jpg)

**Comment by Draise14** [comment]
Subsurface Radius is a cycles setting maybe? OOOOR this prop we split to a vector slider instead of a drop down. It's in the Principled BSDF shader. Maybe it's throwing the addon off.

**Comment by ReinerBforartists** [comment]
I am lost here :)

**Comment by ReinerBforartists** [comment]
Issue in question: Subsurface radius fields are difficult to reach #2318

**Comment by ReinerBforartists** [comment]
Draise you are my hero. The problem was indeed this task where we had renamed N_("Subsurface Radius"), to N_("Subsurface Radius (R,G,B)"), by request.

Lesson learned. NEVER change names in the C code. This will break the API.

The bad news is, i had to revert the name change. The good news is, the addon now works again :)

Fixed in master

![subsurf](https://user-images.githubusercontent.com/13919702/145257243-334942a3-b5e5-47ec-a8d6-b421d2d7a01d.jpg)

![txaworking](https://user-images.githubusercontent.com/13919702/145257260-a605bcc4-55ac-49c1-8744-6c156044997f.jpg)
